### PR TITLE
feat: Paws Loader added with responsive

### PR DESCRIPTION
**Pull request by RamakrushnaBiswal**
## Description
Paws Loader added with responsive
## Related Issue
#469 solved

## Screenshots / GIFs (if applicable)

https://github.com/Innovateninjas/Paws-frontend/assets/125277258/c315c0ae-3e39-4641-8861-cc2387ebf74c

image

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

File: src/App.jsx
Comment (codewarnab) [inline src/App.jsx:1]: you have to cleanup the setTimeout 
```useEffect(() => {
    const timer = setTimeout(() => {
      setLoading(false);
    }, 2500);

    // Cleanup function to clear the timeout
    return () => clearTimeout(timer);
  }, []);
  ```

File: src/Components/shared/loader.jsx
Comment (codewarnab) [inline src/Components/shared/loader.jsx:1]: this component is also being used in the  userprofile page and ngo profile page you have chanaged the props so check if they are working properply or not or any modification needed 
try to add loading for those pages also

**Review by codewarnab — CHANGES_REQUESTED** [review]

File: src/App.jsx
Comment (RamakrushnaBiswal) [reply src/App.jsx:1]: done

File: src/Components/shared/loader.jsx
Comment (RamakrushnaBiswal) [reply src/Components/shared/loader.jsx:1]: solved

**Comment by RamakrushnaBiswal** [comment]
@codewarnab review the PR

File: src/utils/contexts/NgoContext.js
Comment (codewarnab) [inline src/utils/contexts/NgoContext.js:1]: i did not understand why did you returning the component from here ? 
i just told to handle this in profile pages file 
image
its passes a prop which is not in the current loader 
however do not chanage main loader please create seperate loader file which will be shown during first load (app.jsx) load and old loader will be same 
create loader folder their will be two loader file 
also revert this two  context file

**Review by codewarnab — CHANGES_REQUESTED** [review]

File: src/utils/contexts/NgoContext.js
Comment (RamakrushnaBiswal) [reply src/utils/contexts/NgoContext.js:1]: done

**Comment by RamakrushnaBiswal** [comment]
@codewarnab review the PR

**Comment by codewarnab** [comment]
#475

**Pull request closed without merging**